utils/install-protobuf.sh

```diff
@@ -1,6 +1,6 @@
 # Check out protobuf v3.18.3
 PROTOBUF_VERSION=3.18.3
```

Comment:
So is this only used for Mac?

Reply:
@sstamenova Yes!!! Only because Protobuf 3.20.3 failed for Mac, so kept it at the older version.

Reply:
@sstamenova I explain why Mac was failing with Protobuf 3.20.3 here: https://github.com/onnx/onnx-mlir/issues/1936

Reply:
Thanks!